content/es/what-is-opentelemetry.md

```diff
@@ -0,0 +1,132 @@
+  con una amplia variedad de backends de observabilidad, incluyendo
+  [Jaeger](https://www.jaegertracing.io/) y
+  [Prometheus](https://prometheus.io/), así como soluciones comerciales.
+- No es un backend de observabilidad como Jaeger, Prometheus, o productos
+  comerciales de terceros.
+- Enfocado en la generación, recopilación, gestión y exportación de telemetría.
+  Uno de los objetivos principales de OpenTelemetry es que puedas instrumentar
+  fácilmente tus aplicaciones o sistemas, sin importar el lenguaje de
+  programación, infraestructura o entorno de ejecución. El almacenamiento y
+  visualización de la telemetría se deja intencionalmente a otras herramientas.
+
+## ¿Qué es la observabilidad?
+
+[Observabilidad](/docs/concepts/observability-primer/#what-is-observability) es
+la capacidad para comprender el estado interno de un sistema a través del
+análisis de sus resultados. En el contexto del software, esto significa ser
+capaces de comprender el estado interno de un sistema a través del análisis de
+sus datos de telemetría, que incluyen trazas, métricas y logs.
+
+Para que un sistema sea observable, debe ser
+[instrumentado](/docs/concepts/instrumentation), esto es, su código debe emitir
+[trazas](/docs/concepts/signals/traces/),
+[métricas](/docs/concepts/signals/metrics/) o
+[logs](/docs/concepts/signals/logs/). Los datos instrumentados tienen que ser
+enviados luego a un backend de observabilidad.
+
+## ¿Por qué OpenTelemetry?
+
+Con el auge de la computación en la nube, las arquitecturas de microservicios y
+requisitos comerciales cada vez más complejos, la necesidad de
+[observabilidad](/docs/concepts/observability-primer/#what-is-observability) del
+software y la infraestructura es mayor que nunca.
+
+OpenTelemetry satisface la necesidad de observabilidad siguiendo dos principios
+fundamentales:
+
+1. Eres dueño de los datos que generas. No hay ningún tipo de dependencia del
+   proveedor.
+2. Solo tienes que aprender un único conjunto de API y convenciones.
+
+Ambos principios combinados otorgan a los equipos y organizaciones la
+flexibilidad que necesitan en el mundo informático de hoy en día.
+
+Si quieres aprender más, echa un vistazo a
+[la misión, la visión y los valores](/community/mission/) de OpenTelemetry.
+
+## Main OpenTelemetry components
+
+OpenTelemetry consta de los siguientes componentes principales:
+
+- Una [especificación](/docs/specs/otel) para todos los componentes.
+- Un [protocolo](/docs/specs/otlp/) estándar que define el formato de los datos
+  de telemetría.
+- Una [convención semántica](/docs/specs/semconv/) que define un sistema de
+  nomenclatura estándar para tipos de datos de telemetría comunes.
+- APIs que describen cómo se generan los datos de telemetría.
+- [SDKs para cada lenguaje](/docs/languages) que implementan las
+  especificaciones, las APIs, y que exportan datos de telemetría.
+- Un [ecosistema de librerías](/ecosystem/registry) que implementan
```

Comment:
The literal translation of library would be "biblioteca" but both "librería" and "biblioteca" are used in this context

Reply:
I'd rather go with librería, as it's more commonly used I'd say.